File: pset6/sentimental-mario-less/mario.py
from cs50 import get_int

# Keep repromptin till getting integer between 1 - 8
while True:
    h = get_int("Height: ")
    if h <= 8 and h >= 1:
        break

z = 1
s = h-1
for x in range(h):
    # Print spaces
    for k in range(s):
        print(" ", end="")
    # Print hashes
    for y in range(z):
        print("#", end="")
    # Print newline
    print("")
    z += 1
    s -= 1

File: pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    // Get height from user
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Loop for every row
    for (int i = 0; i < height; i++)
    {
        // For every row, print height - row spaces
        for (int j = 0; j < height - 1 - i; j++)
        {
            printf(" ");
        }

        // For every row, print row hashes
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        // For every row, print a new line
        printf("\n");
    }

}